src/check_hooks.h

```diff
@@ -67,6 +67,7 @@ enum error_ids {
 	E_ID_UNKNOWN_PERM      = 7,
 	E_ID_UNKNOWN_CLASS     = 8,
 	E_ID_EMPTY_BLOCK       = 9,
+	E_ID_FC_DUPLICATE_ENTRY = 10,
```

Comment:
I just merged #179 which also added an error check and took E-010.  Can you switch to E-011, please?